iron-doc-behavior.html

```diff
@@ -106,7 +108,8 @@
       },
 
       _noneToShow: function(showProtected, showInherited, descriptor, name) {
-        var items = descriptor[name];
+        var items = name === 'behaviors' ?
+          this._getPolymerBehaviors(descriptor) : descriptor[name];
```

Comment:
I like to format like:

```js
var items = (name === 'behaviors')
    ? this._getPolymerBehaviors(descriptor)
    : descriptor[name];
```

so that it reads like an if/then/else on separate lines